programs/clearing_house/src/controller/orders.rs

```diff
@@ -981,6 +981,21 @@ pub fn fulfill_order_with_amm(
             order_post_only,
         )?;
 
+    let market_postion_unsettled_pnl_delta = cast_to_i128(user_fee)?
```

Comment:
why aren't you using `fee_to_market`?

Reply:
the fee_to_market includes the `quote_asset_surplus` and didn't want to include it in the `unsettled_pnl` since its already within the cost basis